plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/BuildConfigTreeItems.java

```diff
@@ -1,3 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
```

Comment:
I am not sure, if it is ok to have copyright year 2015 :(

Reply:
In commit history, it's 2015, but you're right. If we consider the current changes, 2019 is the more correct year :) Correcting.